.github/workflows/arm-runner.yml

```diff
@@ -8,8 +8,23 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        base_image: ['raspios_lite:2021-10-30', 'dietpi:rpi_armv6_bullseye']
-
+        target: [zero_20211030, zero, zero_dietpi, zero2, zero2_64]
```

Comment:
'target' naming seems a bit irregular.
Shouldn't it be rather something like _platform-os_,
- with platform in (zero, zero2, zero2_64),
- and os in (raspios, dietpi)?

Reply:
@pguyot: Do you intend to support both Bullseye and Pi Zero / Zero 2 in one fell swoop? Quite ambitious! ;-)
This will also yield quite a number of images to build and support...
I guess support for Zero 2 32-bits is not ultimately needed, if Zero 2 64-bits works.

Reply:
Target could also be image suffix or image name. I don't think we need the full 3x2 matrix. dietpi is too different and specialized. Maybe we should simply drop it.

Reply:
> dietpi is too different and specialized. Maybe we should simply drop it.
(I'm may not be fully objective here, since I am the culprit for adding DietPi support in the first place, but) I strongly disagree.
DietPi is not that different, nor _specialized_ (whatever that means), and I have found it more solid and efficient in n number of areas for Pynab support than the base RaspiOS distro.
Also it's easier to use for naive Linux users.